indexing-service/src/main/java/io/druid/indexing/common/task/Task.java

```diff
@@ -49,8 +49,10 @@
     @JsonSubTypes.Type(name = "index_hadoop", value = HadoopIndexTask.class),
     @JsonSubTypes.Type(name = "index_realtime", value = RealtimeIndexTask.class),
     @JsonSubTypes.Type(name = "noop", value = NoopTask.class),
-    @JsonSubTypes.Type(name = "version_converter", value = VersionConverterTask.class),
-    @JsonSubTypes.Type(name = "version_converter_sub", value = VersionConverterTask.SubTask.class)
+    @JsonSubTypes.Type(name = "version_converter", value = SegmentConverterTask.class), // Backwards compat - Deprecated
+    @JsonSubTypes.Type(name = "version_converter_sub", value = SegmentConverterTask.SubTask.class), // backwards compat - Deprecated
+    @JsonSubTypes.Type(name = "segment_convert", value = SegmentConverterTask.class),
```

Comment:
I wonder if we should be more consistent with our naming and start with a verb here like we do for most of our tasks, e.g. simply `convert`, and rename the task `ConvertSegmentTask` accordingly?